operations/template/alert.tf

```
@@ -0,0 +1,43 @@
+resource "azurerm_monitor_action_group" "monitor" {
+  name                = "cdcti${var.environment}-actiongroup"
+  resource_group_name = data.azurerm_resource_group.group.name
+  short_name          = "cdcti-alerts"
+
+  email_receiver {
+    name          = "cdcti-flexion-slack-email-receiver"
+    email_address = var.alert_slack_email
+  }
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "alert" {
```

Let's call this some more descriptive than `alert`.  Perhaps `database_token_expired_alert`.
```suggestion
resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_alert" {
```